gen3/bin/kube-roll-all.sh

```diff
@@ -79,15 +79,15 @@ fi
 
 if g3k_manifest_lookup .versions.fence 2> /dev/null; then
   # data ecosystem sub-commons may not deploy fence ...
-  gen3 kube-setup-fence
+  gen3 kube-setup-fence &
```

Comment:
It it safe to fork some of these setup jobs? I think some apps like fence assume that the audit service ran first to set up the SQS queue, and if things run out of order an app could start with missing resources

Reply:
That's a good point. There must be inter-dependencies between the apps... 
Perhaps the Kubernetes ReadinessProbe should ping the `/status` endpoint and the status check should make sure the other micro-service is up and running -- then the services could be self-aware and we wouldn't need to orchestrate that at the cloud-automation / infra provisioning stage.